lichess-ai-assistant.user.js

```
@@ -339,6 +339,165 @@
+            movesChanged = true;
+            break;
+          }
+        }
+
+        // Check for added/removed move elements
+        if (mutation.type === 'childList') {
+          for (const node of [...mutation.addedNodes, ...mutation.removedNodes]) {
+            if (node.nodeType === Node.ELEMENT_NODE) {
+              // Check if the node is a move element or contains move elements
+              if (node.tagName === 'MOVE' || (node.querySelector && node.querySelector('move'))) {
+                console.log(
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)